src/redux/selectors/getIsUiBlocked.js

```diff
@@ -1,2 +1,8 @@
 export const getIsUiBlocked = state =>
-    state.organisationUnits.loading || state.reportPeriod.loading
+    state.organisationUnits.loading ||
+    state.reportPeriod.loading ||
+    state.dataSet.loading ||
+    (atOrgUnitDistReportSection(state) && state.orgUnitGroupSets.loading)
+
+const atOrgUnitDistReportSection = ({ router }) =>
+    router.location.pathname === '/organisation-unit-distribution-report'
```

Comment:
Please change the string to the config param:
https://github.com/dhis2/reports-app/blob/c9864995a4cb07518d025d67a50e8ae6d9c7fa47/src/config/sections.config.js#L77